packages/cfg/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@syntest/cfg",
```

Comment:
Would it not make more sense to rename this to cfg-core? As otherwise, people might think this package functions be itself